### ISSUE TITLE
Support the U+F8FF character ()

### ISSUE THREAD

**Issue by codefuhrer** [Ranks/emojione#239]
Not sure if this is a bug report or a feauture request.
> [Wikipedia](https://en.wikipedia.org/wiki/Private_Use_Areas#U.2BF8FF):
> In most Apple-supplied fonts, it represents the Apple logo, or an early version of the command key.

So i'll be glad if i can render this symbol via Emojione.

**Comment by caseyahenson** [comment]
@codefuhrer We don't have plans to support specific company logos in our emoji set. This link might provide a little more info on that specific character for you: http://www.fileformat.info/info/unicode/char/f8ff/index.htm

**Comment by codefuhrer** [comment]
Ok, thanks. 
My point was that:
1) Some people uses this symbol as emoji in their texts (despite on that technically it's not an emoji);
2) It doesn't have the emojione support;
3) Webmaster (me) has to parse all the texts one more time – just to replace that symbol.

So consider this issue as feature request :smirk:

**Comment by caseyahenson** [comment]
We definitely appreciate your input on this! Feature request received 👍

**Comment by bet4a** [comment]
😟 I understand the rationale behind this feature request, but am respectfully opposed to its implementation. The `U+F8FF` code point is in the first [Private Use Area](https://en.wikipedia.org/wiki/Private_Use_Areas) block, meaning it is **intentionally** undefined (and is actually *guaranteed* never to be defined as a Unicode character intended for a particular use across different platforms/vendors).

Apple defines `U+F8FF` as the Apple logo, but that's their own **proprietary** usage; other vendors use the same character point (and others in the Private Use Area range) for completely different purposes. On my Chromebook 💻, for example, `U+F8FF` is used to store the precomposed Tibetan glyph ཧྭོ (`U+0F67 U+0FAD U+0F7C`). (Chromebooks actually use [**lots** of Private Use Area](https://sites.google.com/site/chrisfynn2/home/tibetanscriptfonts/standardization/precomposedtibetan-parta/precomposed-tibetan---part-a-page-6) code points to store precomposed Tibetan glyphs.)

Vendors are 🆓 to use any Private Use Area code point for whatever glyph they want… but by definition, such usages are not supposed to be universal. In my opinion, Emoji One should only worry about the emoji characters actually standardized by Unicode. If Emoji One ventures into the Private Use Area, it'd no longer platform-neutral, i.e., it'd be endorsing 🗳☑ how Apple happens to use a Private Use Area code point and enforcing that convention on *everyone* (even those who may be using that particular code point for a completely unrelated purpose, as they should be able to do, since that's what that code block is for!).

**tl;dr**: Emoji One should only implement emojis that have been assigned a universal 🌎 and concrete 📜 meaning by Unicode. Leave brand icons to other tools 🔧—Font Awesome comes to mind 💭; they already have an [Apple logo](http://fortawesome.github.io/Font-Awesome/icon/apple/) glyph available, among many others, which will correctly display as the Apple logo even on non-Apple devices (when implemented correctly).
